modules/sonobiBidAdapter.js

```diff
@@ -323,11 +331,23 @@ function _validateGPID(bid) {
   const gpid = deepAccess(bid, 'ortb2Imp.ext.data.pbadslot') || deepAccess(getGptSlotInfoForAdUnitCode(bid.adUnitCode), 'gptSlot') || bid.params.ad_unit;
 
   if (gpid) {
-    return `|gpid=${gpid}`
+    return `gpid=${gpid},`
   }
   return ''
 }
 
+function _validateMediaType(mediaType) {
+  if (mediaType === 'video') {
+    return 'c=v,';
+  }
+
+  if (mediaType === 'display') {
+    return 'c=d,';
+  }
+
+  return '';
+}
```

Comment:
I would consider writing this as a ternary or setting the values to a variable to avoid the multiple return statements. Something like this:

```
function _validateMediaType(mediaType) {
     return mediaType === 'video' ? 'c=v,' : (mediaType === 'display' ?  'c=d,' : '');
}
```

or 

```
function _validateMediaType(mediaType) {
  let mediaTypeValidation = '';
  if (mediaType === 'video') {
    mediaTypeValidation = 'c=v,';
  } else if (mediaType === 'display') {
     mediaTypeValidation = 'c=d,';
  }

  return  mediaTypeValidation;
}
```